src/openrct2/ride/ride_ratings.c

```diff
@@ -610,32 +610,76 @@ static void ride_ratings_calculate(rct_ride *ride)
 
 static void ride_ratings_calculate_value(rct_ride *ride)
 {
+	typedef struct row {
+		int months, multiplier, divisor, summand;
+	} row;
+
+	static const row age_table_new[] = {
+		{5,         3,          2,          0},   // 1.5x
+		{13,        6,          5,          0},   // 1.2x
+		{40,        1,          1,          0},   // 1x
+		{64,        3,          4,          0},   // 0.75x
+		{88,        9,          16,         0},   // 0.56x
+		{104,       27,         64,         0},   // 0.42x
+		{120,       81,         256,        0},   // 0.32x
+		{128,       81,         512,        0},   // 0.16x
+		{200,       81,         1024,       0},   // 0.08x
```

Comment:
The last row is evaluated differently. 

```
 	// Ride is older than oldest age in the table?
 	if(monthsOld >= last_row.months) {
 		value = (value * last_row.multiplier) / last_row.divisor + last_row.summand;
 	}because it represents the 
```